enacts/config-nma.yaml

```diff
@@ -20,13 +20,18 @@ shapes_adm:
       is_checked: False
 
 # Daily ENACTS Zarr conversion
-rr_mrg_nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/
-rr_mrg_resolution: .25
-rr_mrg_chunks:
+nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/ #top level directory path to find input data
+zarr_resolution: 0 #set to 0 for no regridding
+chunks:
     T: 578
     Y: 25
     X: 31
```

Comment:
Should not the chunks vary with resolution? since we want to strike a balance between number of chunks and their size? and avoid chunks of very different sizes? How to deal with that? Maybe leave a comment about it here so that someone who's going to regrid doesn't forget to think about their chunking?

Reply:
Yes

Reply:
I do think the idea would be to change the chunks at different resolutions - however is it so simple a concept that we are expecting users to know what sizes to be chunking the data? I have not been part of the process for deciding on sizes for chunking so I cannot say for sure, though from what I know so far it seems like something of a case-by-case basis when it comes to deciding on sizes. 

I can at least make a note that says something like: 
"Be aware that changes to resolution may also require different chunking sizes. if regridding, consider updating chunk sizes"

I guess this also brings up the question: should I change the chunk sizes for the Ethiopia maproom, since we are using the native resolution now instead of the regridded data?

Reply:
If it's fast enough, then it's good enough for now. Someday we'll do some experimentation to identify good rules for picking a chunking, and hopefully automate it.

Reply:
For now the "users" are just us really so I think just a comment to remind the next "user" would do. Plus all that is temporary as well as Aaron mentioned.